README.md

```diff
@@ -1,7 +1,11 @@
-***NOTE***: This is a preliminary implementation of the [FIDO Device Onboard Spec](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/) published by the FIDO Alliance. The implementation is experimental and incomplete, and is not ready for use in any production capacity. Some cryptographic algorithms and encoding formats have not been implemented, and any aspect of this implementation is subject to change.
-
 # FIDO Device Onboard (FDO) Protocol Reference Implementation (PRI) Quick Start
 
+This is a reference implementation of the
+[FIDO Device Onboard Spec](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/)
+published by the FIDO Alliance. It provides implementations for the protocol defined by the
+specification. It also provides example implementation for different components to demonstrate
+end-to-end execution of the protocol.
```

Comment:
Do we need to explicitly say it is a production implementation?

Reply:
Updated.